ethpm/package.py

```diff
@@ -12,28 +14,42 @@
     validate_single_matching_uri,
 )
 from ethpm.utils.package_validation import (
-    load_package_data,
     validate_package_against_schema,
-    validate_package_exists,
     validate_package_deployments,
     validate_deployments_are_present,
 )
 
 
+def _load_package_data_from_file(file_obj):
+    try:
+        package_data = json.load(file_obj)
+    except json.JSONDecodeError as err:
+        raise json.JSONDecodeError(
+            "Failed to load package data. File is not a valid JSON document.",
+            err.doc,
+            err.pos
+            )
```

Comment:
This is valid python but we typically would dedent this one more level to be inline with the opening line.  Also, we like trailing commas in python land.